### PR TITLE
WIP: repl enhancements

### PR DESCRIPTION
**Pull request by michaelfig**
Some changes that can be cherry-picked, but mostly hacked-up infix bang support.

### CODE REVIEW

**Comment by michaelfig** [comment]
Relevant PRs for infix bang support in dependencies:

* https://github.com/babel/babel/pull/10115
* https://github.com/Agoric/realms-shim/pull/15
* https://github.com/Agoric/SES/pull/124
* https://github.com/Agoric/SwingSet/pull/85

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ TL;DR: Browse to a public testnet provisioning page, then run:
 $ SOLO_NAME=han ./docker/ag-setup-solo --pull
 ```
 
-where `han` is the same of your solo vat machine that follows the blockchain.
```
Comment (erights) [inline README.md]: Oops. Already fixed in the other PR.

File: package.json [hunk 2]
```diff
@@ -27,6 +30,7 @@
     "minimist": "^1.2.0",
     "morgan": "^1.9.1",
     "node-fetch": "^2.6.0",
+    "ses": "file:ses-0.5.0-infixBang.tgz",
```
Comment (erights) [inline package.json]: How did this previously non have an "ses" dependency?

File: setup/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,10 @@
 ARG TAG=latest
+FROM golang:stretch AS go-build
```
Comment (erights) [inline setup/Dockerfile]: I don't know enough about either docker or golang to review this.

File: setup/ansible/logzio.yml [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
 
-#Note: You need to add LOGZIO_TOKEN variable with your API key. Like tihs: ansible-playbook -e LOGZIO_TOKEN=ABCXYZ123456
+#Note: You need to add LOGZIO_TOKEN variable with your API key.
+# Like this: ansible-playbook -e LOGZIO_TOKEN=ABCXYZ123456 -e JOURNALBEAT_BINARY=/usr/local/bin/journalbeat
 
 - hosts: "{{ service }}"
   user: root
   #any_errors_fatal: true
   gather_facts: no
+  strategy: free
```
Comment (erights) [inline setup/ansible/logzio.yml]: I don't know anything about logzio so I don't know the significance of any of this.

**Comment by michaelfig** [comment]
Closing as all relevant changes have landed on the master branch.

**Pull request closed without merging**